app/views/notifiers/course/forum/post_notifier/replied/user_notifications/email.html.slim

```diff
@@ -7,7 +7,7 @@
 
 - course_title = format_inline_text(course.title)
 - topic_title = format_inline_text(topic.title)
-- post_author = format_inline_text(post.creator.name)
+- post_author = format_inline_text(post_creator_course_user_name(post))
```

Comment:
Can I suggest that instead of using this helper, perhaps we can define a model scopes to find the `course_user` given the `user_id` and `course`. Cleaner, and can be re-used in other places of the codebase.

I can think of 2 more straightforward ways to do this: 
 1. `CourseUser.for(course: course, user: user)`
 2. `course.course_users.for(user_id: user.id)`